client/sites/tools/staging-site/components/staging-site-card/style.scss

```
@@ -0,0 +1,14 @@
+.tools-staging-site__site-url {
+	word-wrap: break-word;
+	text-decoration: none;
+
+	&.is-link {
+		text-decoration: none;
+	}
```

It's redundant, anyway it's teh same selector but we need bigger specificity
```suggestion
	text-decoration: none;
```

---

Good catch, I have fixed it in https://github.com/Automattic/wp-calypso/pull/100762/commits/54e292c5bb0c4ede26f9e68e4685f8e7e393e81b.